pom.xml

```diff
@@ -337,6 +342,11 @@
                 </property>
             </activation>
 
+            <!-- Exclude the example app from releases -->
+            <modules>
+                <module>graphql-kotlin-schema-generator</module>
```

Comment:
it should already be handled by -> https://github.com/ExpediaDotCom/graphql-kotlin/blob/master/graphql-kotlin-spring-example/pom.xml#L81

Reply:
nvm, release runs `nexus-staging-maven-plugin` during deploy phase which was not skipped.